sdk/solana/tbrv3/token-bridge-relayer.ts

```diff
@@ -416,8 +416,9 @@ Current authority: ${upgradeAuthority}`);
 
     return this.program.methods
       .submitOwnerTransferRequest(newOwner)
-      .accounts({
+      .accountsPartial({
         owner: config.owner,
+        tbrConfig: this.account.config().address,
```

Comment:
I'm not understanding why this change is necessary :thinking:

Reply:
I followed the same structure as in the instruction `cancelOwnerTransferRequest`, using `accounts` I had the same error if I change `accountsPartial` for `accounts` on that instruction.

Reply:
For the record, we tried following the type suggested by `acounts` but this resulted in failed tests. Might be some kind of Anchor SDK bug. We will look at this more in depth later.